tests/Mailer/Message/AbstractEventMessageTest.php

```diff
@@ -73,4 +90,24 @@ protected function createCommitteeFeedItemMock(Adherent $author, string $content
 
         return $mock;
     }
+
+    protected function createCitizenActionMock(
+        string $name,
+        string $beginAt,
+        string $street,
+        string $cityCode,
+        $timeZone = \AppBundle\Address\GeoCoder::DEFAULT_TIME_ZONE
```

Comment:
Why not `use AppBundle\Address\GeoCoder;` and call `GeoCoder::DEFAULT_TIME_ZONE` here ?

Reply:
Why not hardcoding `'Europe/Paris'` directly here, instead of using a constant ?

This is a testsuite, we should test expected values also

Reply:
harcoded